### DISCUSSION TITLE
ObjectNav Continuous Action

### DISCUSSION THREAD

**Discussion by AlexandreBrown** [facebookresearch/habitat-lab#2107] [Q&A]
Hi, I would like to know what is a minimally working example of using continuous action space for ObjectNav ? I tried editing the habitat/task/objectnav.yaml without success, my env action space remains discrete.
I notice that the [2023 ObjectNav challenge](https://github.com/facebookresearch/habitat-challenge) is said to have a continuous action space so I would like to know how to use continuous actions from the official habitat-lab repo.

While a previous issue existed for this #165 (comment)
It does not provide a concrete example on how to use continuous actions, only that "it is possible". Therefore I would like a concrete, example if possible.
Thanks!

**Comment by hqm** [comment]
Curious about this also ... anyone working on this?
